src/garage/envs/multi_env_wrapper.py

```diff
@@ -50,18 +52,24 @@ class MultiEnvWrapper(gym.Wrapper):
             A list of objects implementing gym.Env.
         sample_strategy (function(int, int)):
             Sample strategy to be used when sampling a new task.
-
+        metaworld_mt (bool): Set if the envs that are passed Metaworld
```

Comment:
please make this name/description general, rather than referring to metaworld. what does it do, and what does using this flag require of underlying envs?

Reply:
I think I have now added sufficient information

Reply:
Is it possible to remove the metaworld specific logic from this wrapper and call this `add_onehot`? If not, ignore the rest of my comments on this PR.